docs/_includes/api-options.adoc

```diff
@@ -0,0 +1,149 @@
+//= Ruby API Options
+
+// ref issue 562.
+
+.Ruby API options
+[cols="<15m,<35,<20,<20,<25"]
+|====
+|Name |Description |Default value |Allowed values |CLI equivalent
+
+|attributes
+|Passes map of document attributes.
+|_Empty_
+|Select from <<builtin-attributes>>, or user-defined.
+|`-a`, `--attributes`
+
+|backend
+|Output file format
+|html5
+|html5, docbook5, docbook45, manpage
+|`-b`, `--backend`
+
+|base_dir
+|Base directory containing the document and resources.
+|The directory containing the source file, or the working directory if the source is read from a stream.
+|_path_
+|`-B`, `--base-dir`
+
+|catalog_assets
+|?
+|
+|
+|_none_
+
+|compact
+|Not used
+|false
+|true or false
+|_none_
+
+|converter
+|?
```

Comment:
The converter option accepts an instance of a Converter and is used in place of the converter that is automatically resolved from the backend value.

The default value is nil (not set). The allowed value is a Converter instance.